app/server.php

```diff
@@ -83,6 +84,77 @@ function($request, $response) {
         }
     );
 
+App::get('/todos')
+    ->inject('request')
+    ->inject('response')
+    ->action(
+        function($request, $response) {
+            $path = \realpath('/app/app/todos.json');
+            $data = json_decode(file_get_contents($path), true);
+            $response->json($data);
+        }
+    );
+
+App::post('/todos')
+    ->inject('response')
+    ->param('task', "", new Wildcard(), 'Prefs key-value JSON object.')
+    ->param('is_complete', true, new Wildcard(), 'Tells whether task is complete or not')
+    ->action(
+        function($task, $is_complete, $response) {
+            $path = \realpath('/app/app/todos.json');
+            $data = json_decode(file_get_contents($path));
+            $task_entry = ['task' => $task, 'is_complete' => $is_complete];
+            array_push($data, $task_entry);
+            $jsonData = json_encode($data, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);
+            file_put_contents($path, $jsonData);
+            $response->json($jsonData);
+        }
+    );
+
+App::put('/todos/:todo_name')
```

Comment:
```suggestion
App::put('/todos/:id')
```

Let's not use the todo_name to identify a todo.